### PR TITLE
refactor(frontend): Extract constants for svelte slide transition

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

To avoid repetition, we extract the constants that define the Svelte transition `slide`, since they are used a few times in the code.

### CODE REVIEW

File: src/frontend/src/lib/constants/transition.constants.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { quintOut } from 'svelte/easing';
+import type { SlideParams } from 'svelte/transition';
+
+export const SLIDE_DURATION = { duration: 250 };
+
+export const SLIDE_QUINT_OUT: Pick<SlideParams, 'easing' | 'axis'> = {
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/lib/constants/transition.constants.ts:6]: undecided between `SLIDE_QUINT_OUT` and `SLIDE_EASING`

File: src/frontend/src/lib/constants/transition.constants.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { quintOut } from 'svelte/easing';
+import type { SlideParams } from 'svelte/transition';
+
+export const SLIDE_DURATION = { duration: 250 };
```
Comment (peterpeterparker) [inline src/frontend/src/lib/constants/transition.constants.ts:4]: Can you set the TS type? `SlideParams` or something

File: src/frontend/src/lib/constants/transition.constants.ts (hunk 1 above)
Comment (peterpeterparker) [reply src/frontend/src/lib/constants/transition.constants.ts:6]: No strong opinion

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**